src/main/java/no/nav/veilarbvedtaksstotte/domain/VedtakOpplysningKilder.kt

```diff
@@ -3,5 +3,6 @@ package no.nav.veilarbvedtaksstotte.domain
 enum class VedtakOpplysningKilder(val desc: String) {
     CV("CV-en"),
     REGISTRERING("Svarene dine fra da du registrerte deg"),
-    EGENVURDERING("Svarene dine om behov for veiledning")
+    EGENVURDERING("Svarene dine om behov for veiledning"),
+    ARBEIDSSOKERREGISTERET("Svarene dine fra registrering som arbeidssøker"),
```

Comment:
Vises disse tekstene noe sted? 
For å matche de andre så burde den vel være:
```suggestion
    ARBEIDSSOKERREGISTERET("Svarene dine fra da du registrerte deg som arbeidssøker"),
```
Men det er jo samtidig ganske langt, så eventuelt "Svarene dine fra din registrering som arbeidssøker"? (som jo også er relativt langt da)

Reply:
Kan ta en dobbelsjekk med Kari 😊